api/analysis/analysis.go

```diff
@@ -161,3 +161,55 @@ func monitorAnalysisCheckStatus(RID string) (bool, error) {
 	}
 	return analysisFinished, err
 }
+
+func updateInfoAndResultBasedOnCID(cInfo, cResult, CID string) error {
+
+	updateContainerAnalysisQuery := bson.M{
+		"$set": bson.M{
+			"containers.$.cResult": cResult,
+			"containers.$.cInfo":   cInfo,
+		},
+	}
+
+	analysisQuery := map[string]interface{}{"containers.CID": CID}
+	err := db.UpdateOneDBAnalysisContainer(analysisQuery, updateContainerAnalysisQuery)
+	if err != nil {
+		log.Error("updateCIDinfoAndResult", "ANALYSIS", 2007, err)
```

Comment:
This log description does not match the function name. It can be:
```
log.Error("updateInfoAndResultBasedOnCID", "ANALYSIS", 2007, err)
```

Reply:
Good, catch! 😅 I have just fixed it here at 7f57164.